aperturedb/PytorchData.py

```diff
@@ -4,6 +4,21 @@
 
 
 class PytorchData(Subscriptable):
+    """
+    **Class to wrap around a Dataset retrieved from torchvision.datasets**
```

Comment:
Do we have a link to refer here for torch vision?

Reply:
will add